zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/actuator/SetFlowControlActuator.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.qa.util.actuator;
+
+import feign.Body;
+import feign.Feign;
+import feign.Headers;
+import feign.Param;
+import feign.RequestLine;
+import feign.Retryer;
+import feign.Target.HardCodedTarget;
+import feign.jackson.JacksonEncoder;
+import io.camunda.zeebe.qa.util.cluster.TestApplication;
+import io.zeebe.containers.ZeebeNode;
+
+public interface SetFlowControlActuator {
```

Comment:
:wrench: No need to define separate actuator clients for get & set. Just have one `FlowControlActuator`.

Reply:
Here I used different actuators because I have a different decoder for each of the set and get actuators, and therefore use a different feign client. Maybe there is a easier way to use different configs in the client but I am not sure 🤔

Reply:
Isn't the only difference that here you don't set a decoder? Does that really matter?

Reply:
Yes, in this case using the decoder or not will fail to parse one or vice versa.

Reply:
Now with the refactor of the adminRequest it was already possible to just have one Feigh actuator :)